config/EnderStorage.cfg

```diff
@@ -8,7 +8,7 @@ anarchy-mode=false
 checkUpdates=false
 
 #Set to true to make the vanilla enderchest unplaceable.
-disable-vanilla=true
+disable-vanilla=false
```

Comment:
You can found enderchests in the End so far i know. If you enable it The enderchest mod not working any more.